### ISSUE TITLE
[Feature Request] Distributions: reverse calculation

### ISSUE THREAD

**Issue by Franck6S** [jasp-stats/jasp-issues#2646]
### Description

When making a calculation of a domain, JASP displayed only 2 digits where I have a need for a minimum of 4. and when I wnat toi generate the reverse calculation it does not exist a way to perform it

### Purpose

calculation of partof population with enough precision in both ways

### Use-case

We are applying that activity in our calculation activities (was in use without problem with Minitab)

### Is your feature request related to a problem?

Being able to get enough digit, not possible with adjustement of parameter digit into the graphical option

### Is your feature request related to a JASP module?

Distributions

### Describe the solution you would like

Generate a table below the graph with the full value 5 or more representative digits

### Describe alternatives that you have considered

We are making a seperate calculation into excel for the time-being, Should be more valuable to get all in JASP

### Additional context

![distribution domain 1](https://github.com/jasp-stats/jasp-issues/assets/152309774/384b51df-aa41-4fad-8717-06c1288c7800)

**Comment by Franck6S** [comment]
Following the sequence in several ways illustrated with the excel calculation (including reverse way):
![distribution domain 1 example 1_3](https://github.com/jasp-stats/jasp-issues/assets/152309774/aad7e5a9-3414-43c2-9f14-3d93d88be11a)
![distribution domain 1 example 2_3](https://github.com/jasp-stats/jasp-issues/assets/152309774/cdfd0c6d-f36a-41bb-b297-63d52001ddef)
![distribution domain 1 example 3_3](https://github.com/jasp-stats/jasp-issues/assets/152309774/b8d41342-f9e5-4b93-aa3c-b453db77417e)

**Comment by boutinb** [comment]
There exists already in Preferences/Results/Table Options the "Fix the number of decimals" option. We could use this or add another option special for the plots.
But when I have a quick look at the R code in the modules, the round function is used many times with a hard-coded 2 or 3 decimals. So we would have to replace all these occurrences in all modules. We could start by the Distributions module. I have experimented quickly, this is quite easy to do...
Any thoughts @Kucharssim , @vandenman ?

**Comment by Franck6S** [comment]
Dears, It was already tried, without success, see screenshot of current situation 
![image](https://github.com/jasp-stats/jasp-issues/assets/152309774/1a70c890-f3e6-4576-bb27-dc9f1b530fe4)
The other option that I could propose is to add a table below the graph with the results

**Comment by Franck6S** [comment]
Regarding the reverse calculation meaning I want to know how the limit to get 25%. (see last slide) I suppose another development is required to get 0.674488 ?

**Comment by EJWagenmakers** [comment]
The graph would look weird if it had very many decimals. The preferred solution is to provide a *table* that summarizes the numbers shown in the figure.

**Comment by tomtomme** [comment]
@Franck6S 
Seems to be a duplicate of
https://github.com/jasp-stats/jasp-issues/issues/1597

Do you agree?

**Comment by JTPetter** [comment]
I Just spoke to Franck and wanted to know if there are any updates on this and the request for more decimals @Kucharssim?

**Comment by Kucharssim** [comment]
Yes it's on my list, but I haven't gotten to update the module in a while. Hopefully soon

**Comment by Franck6S** [comment]
Please let me know what is your plan ? Could I expect it updated by April ?

**Comment by tomtomme** [comment]
@Franck6S 
April might be ambitious. We have 400 feature requests here :D
Please be aware, that most of our analysis-developers push these improvements in their free time.

**Comment by bensagalovsky** [comment]
As @tomtomme linked above, I entered a specific request for the inverse calculation, without knowing how this issue had evolved. This is the way that I phrased it:

_In Distribution, being able to input a cumulative or tail probability and getting the corresponding x value, as well as shading the corresponding area of the density plot._

This would be quite useful when teaching - for illustrating confidence intervals, for instance, as well as critical values, even though this last concept is a bit outdated today - and for estimating for instance the time till a given percent of failures. 

Thanks!
